### PR TITLE
Repro_IoTOperations_DO-NOT-MERGE

### PR DESCRIPTION
**Pull request by welovej**
# Contributing to the Azure SDK

Please see our [CONTRIBUTING.md](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md) if you are not familiar with contributing to this repository or have questions.

For specific information about pull request etiquette and best practices, see [this section](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md#pull-request-etiquette-and-best-practices).

### CODE REVIEW

File: sdk/iotoperations/ci.mgmt.yml [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+pr:
+  branches:
+    include:
+    - main
+    - feature/*
+    - hotfix/*
+    - release/*
+  paths:
+    include:
+    - sdk/ProviderNameLowercase/ci.mgmt.yml
```
Comment (mmorrison57) [inline sdk/iotoperations/ci.mgmt.yml:13]: Are these expected to say `ProviderNameLowercase` and not the actual service name?

File: sdk/iotoperations/Azure.ResourceManager.IoTOperations/src/Azure.ResourceManager.IoTOperations.csproj [hunk 2]
```diff
@@ -0,0 +1,8 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <Version>1.0.0-beta.1</Version>
+    <PackageId>Azure.ResourceManager.IoTOperations</PackageId>
+    <Description>Azure Resource Manager client SDK for Azure resource provider IoTOperations.</Description>
+    <PackageTags>azure;management;arm;resource manager;ProviderNameLowercase</PackageTags>
```
Comment (mmorrison57) [inline sdk/iotoperations/Azure.ResourceManager.IoTOperations/src/Azure.ResourceManager.IoTOperations.csproj:6]: Same question here - if this should be the actual service name or not

File: sdk/iotoperations/ci.mgmt.yml (hunk 1 above)
Comment (welovej) [reply sdk/iotoperations/ci.mgmt.yml:13]: yes, you are right. I will quickly fix this. 
And this is a new repro [PR ](https://github.com/Azure/azure-sdk-for-net/pull/45916)with the issue you mentioned fixed.

File: sdk/iotoperations/Azure.ResourceManager.IoTOperations/src/Azure.ResourceManager.IoTOperations.csproj (hunk 2 above)
Comment (welovej) [reply sdk/iotoperations/Azure.ResourceManager.IoTOperations/src/Azure.ResourceManager.IoTOperations.csproj:6]: yes, you are right. I will quickly fix this. 
And this is a new repro [PR ](https://github.com/Azure/azure-sdk-for-net/pull/45916)with the issue you mentioned fixed.

**Pull request closed without merging**